### ISSUE TITLE
MZDY Nepeňažné plnenie príjem

### ISSUE THREAD

**Issue by 61Stan** [eurosecom/projekt1#82]
Vo výkazoch pre zdravotné poisťovne nemôže ísť nepeňažné plnenie do hrubého príjmu zamestnanca, čiže sa musí rovnať vymeriavací základ s hrubým príjmom.

**Comment by eurosecom** [comment]
Ak má zamestnanec nepeňažné plnenie podla zákona 595/2013 o dani z prijmu § 8
Ostatné príjmy
ods. (1)
písmeno
l) peňažné a nepeňažné plnenie, ktoré bolo poskytnuté poskytovateľovi zdravotnej starostlivosti,37aa) jeho zamestnancovi alebo zdravotníckemu pracovníkovi 
od držiteľa registrácie lieku, držiteľa povolenia na veľkodistribúciu liekov, držiteľa povolenia na výrobu liekov, farmaceutickej spoločnosti,37ab) 
výrobcu zdravotníckej pomôcky, výrobcu dietetickej potraviny37ac) alebo prostredníctvom tretej osoby (ďalej len „držiteľ“),

nenahrávame do miezd lebo sa jedná o zrážkovú daň, na ktorú sa vzťahuje špeciálna oznamovacia povinnosť a platí ju zamestnanec sám za seba na svoj daňovým úradom pridelený účet. Do konca mesiaca keď mal tento príjem musí oznámiť DÚ, že mal tento príjem, neviem ako, nemám tlačivo. Do konca mesiaca po skončení štvrťroka , teda do 30.4:2015 za 1. až 3. mesiac musí podať oznámenie o príjmch a zrážkovej dani, tlačivo je v následujúcom commite a bolo by dobré vytvoriť formulár, bude sa týkať lekární a zdravotníckych zariadení asi aj DSS.

**Comment by eurosecom** [comment]
v selectore vyber osc aj ico na konci

**Comment by eurosecom** [comment]
este daj vyhladavacie tlacidlo k adrese zdravotnickeho zariadenia a k drzitelovi aby sa to dalo kopirovat

**Comment by 61Stan** [comment]
Zverejnili online formulár https://pfseform.financnasprava.sk/Formulare/eFormVzor/DP/form.334.html - poprosím základnú štruktúru xml

**Comment by 61Stan** [comment]
Preveril som s 5 držiteľmi a divne majú spravené pri stránkovaní.
Pri tlači: Počet priložených strán = 3(na 2.strane). V prílohe je len 1 strana, kde v záhlaví je 1/1 (3.strana oznámenia).
Pri xml: Počet priložených strán = 3(na 2.strane). V prílohe sú držitelia ako zvlášť strana, čiže pred každým <drzitel> je <aktualna>1</aktualna> až 5 z <celkovo>5</celkovo>
Testovaný súbor zakomponujem do .xml

**Comment by 61Stan** [comment]
Stav bez držiteľa nastane, pretože "Vypracoval" a jeho telefón generujeme z údajov o firme.
Stránkovanie vôbec nie je OK. Keď si urobím oznámenie s 5 držiteľmi, tak mi načíta do edane java len jedného. Spôsobené tým, že každým elementom držiteľa musí byť v <priloha> a pred <drzitel> musí byť element <strana> a v nej <aktualna> a <celkovo>. Všetko je to vidieť v "oznamenie_zrdxml2015.php" vo vzore na začiatku skriptu. Element <celkovo> má byť 5 a nie je 1 (ak mám 5 držiteľov).
Na 2.strane počet priložených oni dávajú 3 (ak sú 5 držitelia) = to asi majú chybu, pretože v prílohe je strana 1/1.

Číslo telefónu vo vypracoval musí byť bez lomítka.

**Comment by eurosecom** [comment]
Ak nie je drzitel, nie je prijem, nieje dan -> nepodavam priznanie, = stav bez drzitela NENASTANE

**Comment by eurosecom** [comment]
ten vzor XML co si poslal je vygenerovanych 5 uzivatelov z edane ?

**Comment by 61Stan** [comment]
Áno, vyexportované z edane java. Ale budú asi prerábať, je problém aj s tlačou výkazu.

**Comment by 61Stan** [comment]
Načítanie držiteľov (skúšal som na 5) prebehlo OK.
